alf/examples/sac_pendulum.gin

```diff
@@ -0,0 +1,44 @@
+
+import alf.algorithms.sac_algorithm
+import alf.utils.common
+
+# environment config
+NUM_PARALLEL_ENVIRONMENTS=1
```

Comment:
why only use 1 here?

Reply:
Its an appropriate value here .  process communication  would be heavy load for this simple gym env when `ParallelPyEnvironment` is used ,it actually cost less training steps when parallel environments are used but it slows down the training it terms of total time